.github/workflows/documentation.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy Documentation to GitHub Pages
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - master
```

Comment:
did you mean main?

Reply:
Yes, copy/paste error from different repo where this script was live.